jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/resource/FileSystemResourceTest.java

```diff
@@ -1143,21 +1134,24 @@ public void testResolveWindowsSlash() throws Exception
         try
         {
             assertThat("Exists: " + basePath, base.exists(), is(true));
-            assertThat("Target URI: " + basePath, base, hasNoTargetURI());
+            assertThat("Target URI: " + basePath, base, isNotAlias());
 
             Resource r = base.resolve("aa%5C/foo.txt");
-            assertThat("getURI()", r.getPath().toString(), containsString("aa\\/foo.txt"));
-            assertThat("getURI()", r.getURI().toASCIIString(), containsString("aa%5C/foo.txt"));
-
             if (org.junit.jupiter.api.condition.OS.WINDOWS.isCurrentOs())
             {
+                assertThat("getURI()", r.getPath().toString(), containsString("aa\\foo.txt"));
```

Comment:
This test was already failing on 12.0.x branch when run on windows. 
So this isn't directly related to the other changes.

Reply:
Why is it `getURI()` and then using path to validate the string?